app/script/event/WebApp.coffee

```diff
@@ -109,8 +108,16 @@ z.event.WebApp =
   LEFT:
     HIDE: 'wire.webapp.left.hide'
     FADE_IN: 'wire.webapp.left.fade-in'
+  LIFECYCLE:
+    ASK_TO_CLEAR_DATA: 'wire.webapp.lifecycle.ask-to-clear-data'
+    LOADED: 'wire.webapp.lifecycle.loaded'
+    REFRESH: 'wire.webapp.lifecycle.refresh'
+    RESTART: 'wire.webapp.lifecycle.restart'
+    SIGN_OUT: 'wire.webapp.lifecycle.sign_out'
```

Comment:
Unfortunately not. It's a mix of both in the file. `sign_out` has the upside of marking the full string part on double-click.